CHANGELOG.md

```diff
@@ -2,10 +2,349 @@
+  [#4153](https://github.com/mixxxdj/mixxx/pull/4153)
+* Refactoring of beats object
+  [#4044](https://github.com/mixxxdj/mixxx/pull/4044)
+  [#4048](https://github.com/mixxxdj/mixxx/pull/4048)
+  [#4045](https://github.com/mixxxdj/mixxx/pull/4045)
+  [#4049](https://github.com/mixxxdj/mixxx/pull/4049)
+  [#4092](https://github.com/mixxxdj/mixxx/pull/4092)
+  [#4094](https://github.com/mixxxdj/mixxx/pull/4094)
+  [#4104](https://github.com/mixxxdj/mixxx/pull/4104)
+  [#4103](https://github.com/mixxxdj/mixxx/pull/4103)
+  [#4127](https://github.com/mixxxdj/mixxx/pull/4127)
+  [#4099](https://github.com/mixxxdj/mixxx/pull/4099)
+  [#4071](https://github.com/mixxxdj/mixxx/pull/4071)
+  [#4184](https://github.com/mixxxdj/mixxx/pull/4184)
+* Refactoring of audio engine code
+  [#2762](https://github.com/mixxxdj/mixxx/pull/2762)
+  [#2801](https://github.com/mixxxdj/mixxx/pull/2801)
+  [#2885](https://github.com/mixxxdj/mixxx/pull/2885)
+  [#2997](https://github.com/mixxxdj/mixxx/pull/2997)
+  [#3266](https://github.com/mixxxdj/mixxx/pull/3266)
+  [#4064](https://github.com/mixxxdj/mixxx/pull/4064)
+  [#4065](https://github.com/mixxxdj/mixxx/pull/4065)
+  [#4066](https://github.com/mixxxdj/mixxx/pull/4066)
+  [#4069](https://github.com/mixxxdj/mixxx/pull/4069)
+  [#4074](https://github.com/mixxxdj/mixxx/pull/4074)
+  [#4075](https://github.com/mixxxdj/mixxx/pull/4075)
+  [#4076](https://github.com/mixxxdj/mixxx/pull/4076)
+  [#4078](https://github.com/mixxxdj/mixxx/pull/4078)
+  [#4082](https://github.com/mixxxdj/mixxx/pull/4082)
+  [#4077](https://github.com/mixxxdj/mixxx/pull/4077)
+  [#4080](https://github.com/mixxxdj/mixxx/pull/4080)
+  [#4086](https://github.com/mixxxdj/mixxx/pull/4086)
+  [#4089](https://github.com/mixxxdj/mixxx/pull/4089)
+  [#4090](https://github.com/mixxxdj/mixxx/pull/4090)
+  [#4079](https://github.com/mixxxdj/mixxx/pull/4079)
+  [#4091](https://github.com/mixxxdj/mixxx/pull/4091)
+  [#4083](https://github.com/mixxxdj/mixxx/pull/4083)
+  [#4095](https://github.com/mixxxdj/mixxx/pull/4095)
+  [#4081](https://github.com/mixxxdj/mixxx/pull/4081)
+  [#4061](https://github.com/mixxxdj/mixxx/pull/4061)
+  [#4105](https://github.com/mixxxdj/mixxx/pull/4105)
+  [#4183](https://github.com/mixxxdj/mixxx/pull/4183)
+  [#4186](https://github.com/mixxxdj/mixxx/pull/4186)
+
+### Controller ###
+* Allow empty Control Objects [#2947](https://github.com/mixxxdj/mixxx/pull/2947)
+* Update Novation Launchpad controller scripts [#2600](https://github.com/mixxxdj/mixxx/pull/2600)
+* Add generic USB HID "Set Reports (Feature)" functionality [#3051](https://github.com/mixxxdj/mixxx/pull/3051)
+* HID functions to read back the status of an HID controller (like MIDI SYSEX) [#3317](https://github.com/mixxxdj/mixxx/pull/3317)
+* Use hidraw backend of hidapi instead of libusb backend [#4054](https://github.com/mixxxdj/mixxx/pull/4054)
+* Rename Preset to Mappings [#3472](https://github.com/mixxxdj/mixxx/pull/3472)
```

Comment:
```suggestion
* Consistently use "mapping" instead of "preset" to refer to controller mappings [#3472](https://github.com/mixxxdj/mixxx/pull/3472)
```